.github/workflows/deploy_production.yml

```diff
@@ -39,6 +39,8 @@ jobs:
       app-s3-region: "ap-southeast-1"
       app-s3-assets-bucket-name: "isomer-next-infra-prod-assets-private-a319984"
       app-s3-assets-domain-name: "isomer-user-content.by.gov.sg"
+      app-growthbook-client-key: "sdk-r07MHTLLgfdVDThi"
+      app-intercom-app-id: "jv2tjc3g"
```

Comment:
will the leading and trailing double quotes cause breaking changes?

Reply:
Hmm I don't think so, because this is a string input into the workflow, and the interpolation inside aws_deploy.yml will remove the quotes.